docs/products/metrics/howto.md

```diff
@@ -0,0 +1,9 @@
+---
+title: HowTo
+---
+
+Get started with Aiven for Metrics in your own projects.
```

Comment:
You're setting your content for SEO competition with this PR https://github.com/aiven/aiven-docs/pull/148/files

Could you update this line since it's not about getting started?